### DISCUSSION TITLE
The Offering App - empty project for propagateDependencies

### DISCUSSION THREAD

**Discussion by christianbraeunlich** [microsoft/alguidelines#217] [BC Best Practices]
Hello there,
after reading https://www.waldo.be/2022/05/13/microsoft-dynamics-365-business-central-offering-apps/ I was wondering if it's a Best Practice to create an "empty" app that only holds the dependencies and the manifest (app.json) with the propagateDependencies key set to true. This is also how Microsoft is doing it with the "Application" that includes the System and the Base.

Let's discuss! 🦦

**Comment by waldo1001** [comment]
Well - for "offering apps", I think it's indeed a good practice 🤪

**Reply by christianbraeunlich** [reply]
> Well - for "offering apps", I think it's indeed a good practice 🤪

Would the creator of this Best Practice like to describe it? 🦦

**Reply by waldo1001** [reply]
Lol - would love to - when I have the time .. 🤔.

**Reply by christianbraeunlich** [reply]
..we need this..
![image](https://user-images.githubusercontent.com/25108568/215763961-843d622e-4f38-48bf-a764-537708cc8181.png)
